Femlives/femlives
# Feat/fl 24/convex setup

## Description

Sets up Convex and advanced starting `npm run dev` handling

## How Can This Be Tested?

We have to test this but it could be that there needs to be done a login to work with the server
-> You can try the new `npm run dev` command and see what happens

!!! IMPORTANT !!!
There are new entries in our lenv.local documentation i notion which you have to add

## Checklist

- [x] 🙅‍♀️ No leftover console.logs
- [ ] 🦾 Sufficient tests, e. g. unit test
- [x] 🧠 Left over TODOs are marked with a url to the ticket
